pkg/authenticator/version.go

```diff
@@ -0,0 +1,15 @@
+package authenticator
+
+import "fmt"
+
+// Version field is a SemVer that should indicate the baked-in version
+// of the authn-k8s-client
+var Version = "0.16.1"
+
+// Tag field denotes the specific build type for the client.
+var Tag = "dev"
```

Comment:
Please change comment to:
```
// Tag field denotes the specific build type for the broker. It may
// be replaced by compile-time variables if needed to provide the git
// commit information in the final binary. See `Static long version tags`
// in the `Building` section of `CONTRIBUTING.md` for more information on
// this variable.
```